docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -97,12 +97,11 @@ the algorithm extracts list of masked detector IDS from source workspace and
 uses them to mask the corresponding spectra of the target workspace. 
 
 Setting property *ForceInstrumentMasking* to true forces algorithm 
-to always use *MaskedWorkspace* detector IDs
-as the source of the masking information. 
-If a detector is masked, then the corresponding detector
+to always use *MaskedWorkspace* detector IDs as the source of the masking
+information. If a detector is masked, then the corresponding detector
 will be masked in the input *Workspace*.
 
-If the input *MaskedWorkspace* is a `Matrix Workspace <http://docs.mantidproject.org/nightly/concepts/MatrixWorkspace.html#matrixworkspace>`_ 
+If the input *MaskedWorkspace* is a :ref:`Matrix Workspace <MatrixWorkspace>` 
```

Comment:
Put "MaskedWorkspace" in double back-ticks.

Reply:
Can I inquire what for you suggest this change? All the other properties in this `.rst` file are decorated with `*`, including all occurrences of "MaskedWorkspace".

Reply:
I've been slowly changing algorithm properties to double back-ticks as I find them. Since I was looking at the diff, I didn't see it elsewhere in the file. This change can be skipped.

Reply:
Ok, I understand now. I think the change is out of the scope of this PR as there are quite a few properties around this file that would need changing.